FBSDKLoginKit/FBSDKLoginKit/FBSDKLoginManager.m

```diff
@@ -741,7 +741,7 @@ - (void)performWebLogInWithParameters:(NSDictionary *)loginParams handler:(void(
                                                            [FBSDKInternalUtility bundleForStrings],
                                                            @"Log In",
                                                            @"Title of the web dialog that prompts the user to log in to Facebook.");
-  [FBSDKWebDialog showWithName:@"oauth" parameters:loginParams delegate:self];
+  [FBSDKWebDialog showWithName:@"oauth" parameters:parameters delegate:self];
```

Comment:
New dictionary is created at https://github.com/facebook/facebook-ios-sdk/pull/836/files#diff-759dce5bd7cb70780cf5470f82900ae4R738 but not used. Maybe missing title info.